Vocable/Common/ItemSelection.swift

```diff
@@ -10,20 +10,16 @@ import Foundation
 import Combine
 import CoreData
 
-struct ItemSelection {
+class ItemSelection {
```

Comment:
```suggestion
struct ItemSelection {
```
I think this can go back to being a struct with the new property wrapper